pkg/trace/stats/concentrator.go

```diff
@@ -55,37 +58,36 @@ type Concentrator struct {
 	statsd                 statsd.ClientInterface
 }
 
-var defaultPeerTags = []string{
-	"_dd.base_service",
-	"amqp.destination",
-	"amqp.exchange",
-	"amqp.queue",
-	"aws.queue.name",
-	"bucketname",
-	"cassandra.cluster",
-	"db.cassandra.contact.points",
-	"db.couchbase.seed.nodes",
-	"db.hostname",
-	"db.instance",
-	"db.name",
-	"db.system",
-	"hazelcast.instance",
-	"messaging.kafka.bootstrap.servers",
-	"mongodb.db",
-	"msmq.queue.path",
-	"net.peer.name",
-	"network.destination.name",
-	"peer.hostname",
-	"peer.service",
-	"queuename",
-	"rpc.service",
-	"rulename",
-	"server.address",
-	"statemachinename",
-	"streamname",
-	"tablename",
-	"topicname",
-}
+//go:embed peer_tags.ini
+var peerTagFile []byte
+
+var defaultPeerTags = func() []string {
+	cfg, _ := ini.Load(peerTagFile)
+	keys := cfg.Section("dd.apm.peer.tags").Keys()
+
+	var tags []string = []string{"_dd.base_service", "cassandra.cluster",
+		"hazelcast.instance", "rulename", "statemachinename"}
+
+	if len(keys) > 0 {
+		for _, key := range keys {
+			value := strings.Split(key.Value(), ",")
+			for _, tag := range value {
+				if tag != "grpc.host" && tag != "http.host" &&
+					tag != "cassandra.keyspace" && tag != "rpc.system" &&
+					tag != "messaging.system" && tag != "peer.cluster.name" &&
+					tag != "hostname" && tag != "out.host" &&
+					tag != "messaging.destination" && tag != "messaging.destination.name" &&
+					tag != "messaging.rabbitmq.exchange" {
+					tags = append(tags, tag)
+				}
+			}
+		}
+	}
```

Comment:
This doesn't make sense to me. Why are we defining the tags in a file, but then hardcoding all of these defaults and exceptions??